libc/test/src/__support/arg_list_test.cpp

```diff
@@ -72,9 +72,9 @@ long int check_primitives(int first, ...) {
   count += args.next_var<unsigned long>();
   count += args.next_var<long long>();
   count += args.next_var<unsigned long long>();
-  count += args.next_var<double>();
-  count += args.next_var<double>();
-  count += args.next_var<long double>();
+  count += static_cast<long>(args.next_var<double>());
+  count += static_cast<long>(args.next_var<double>());
+  count += static_cast<long>(args.next_var<long double>());
```

Comment:
Mind changing the return type and type of `count` to just `long`?  `int` is implied, and isn't specified generally throughout llvm.

Same for `check_struct_type` below.